app/src/main/java/com/example/newsportal/data/remote/NewsService.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.newsportal.data.remote
+
+import com.example.newsportal.data.remote.model.NewsResponce
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface NewsService {
+
+    companion object{

```

Comment:
companion object should be placed at the bottom of the class according to kotlin conversions